code/modules/client/client_procs.dm

```diff
@@ -634,6 +634,7 @@ GLOBAL_LIST_INIT(blacklisted_builds, list(
 	qdel(query_log_connection)
 
 	SSserver_maint.UpdateHubStatus()
+	SSambience.ambience_listening_clients -= src
```

Comment:
Are you sure this is the right place, instead of on `Destroy()` ?

Reply:
it definitely is not, but what the hell is this proc